app/views/claims/information_provided.html.erb

```diff
@@ -0,0 +1,48 @@
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+
+    <h1 class="govuk-heading-xl">How we will use the information you provide</h1>
+    <p class="govuk-body">
+      To process your claim we will check the information you provide against the information the
+      Department for Education hold.
+    </p>
+
+    <p class="govuk-body">To determine if your claim is valid we will:</p>
+
+    <ul class="govuk-list govuk-list--bullet">
+      <li>
+        check your eligibility against the Database of Qualified Teachers. You may have used with
+        this in the past as the Teacher Self-Service Portal
```

Comment:
Is it weird to have multiple sentences in these? The list items start with lowercase, then have a full stop, then end without one. Just looks weird to me and is something I always struggle to avoid.

Reply:
Yes, this content has been put together quite hastily and we expect it to change once Patrick has given it the once over. Bulleted lists should ideally be single sentence, no full stop, but because it is two sentences for now, we should end this with a full stop.